opennms-doc/guide-admin/src/asciidoc/text/telemetryd/troubleshooting.adoc

```diff
@@ -0,0 +1,57 @@
+
+[[telemetryd-troubleshooting]]
+=== Troubleshooting
+
+Configuring _telemetryd_ involves a few components.
+This section gives a few troubleshooting hints to verify if the configuration and the network setup is correct.
+
+.Ensure the Java process is listening on a specific port
+[source]
+----
+ss -lnp sport = :8877
+
+Netid State  Recv-Q Send-Q Local Address:Port Peer Address:Port
+udp   UNCONN 0      0      *:8877                           *:* users:(("java",pid=585,fd=1226))
+----
+
+.Ensure Firewall is not blocking incoming traffic
+[source]
+----
+sudo iptables -L
+----
+
+.Example to set a firewall rule with firewalld for 8877/udp
+[source]
+----
+firewall-cmd --add-port=8877/udp --zone=public
+----
+or
+
+.Example to set a firewall rule with iptables for 8877/udp
+[source]
+----
+iptables -I INPUT -p udp -m udp --dport 8877 -j ACCEPT
+----
+
+.Check if packets can be written to the internal queue for the specific protocol `Netflow-5`
+[source]
+----
+tail -f ${OPENNMS_HOME}/logs/telemetryd.log | grep "JmsConsumer[OpenNMS.Sink.Telemetry.Netflow-5]"
+----
+
+If you see errors like
+
+[source]
+----
+org.springframework.jms.JmsSecurityException: User minion is not authorized to write to: queue://OpenNMS.Sink.Telemetry.Netflow-5; nested exception is javax.jms.JMSSecurityException: User minion is not authorized to write to: queue://OpenNMS.Sink.Telemetry.Netflow-5
```

Comment:
I'd opt for documenting this in a bug report instead of here.

Reply:
This is not related to the problem I have encountered with ROLE_MINON / ROLE_ADMIN issue. Can this also happen if a user forgets to assign the role during a minion setup?

Reply:
Maybe, but this type of problem will also affect the Heartbeat, Sylogs forwarding, Trap forwarding, etc... It seems odd to mention this in the specific context of Telemetryd.